src/context/WalletProvider/WalletProvider.tsx

```diff
@@ -410,12 +403,13 @@ export const WalletProvider = ({ children }: { children: React.ReactNode }): JSX
     const localWalletDeviceId = getLocalWalletDeviceId()
     if (localWalletType && localWalletDeviceId) {
       ;(async () => {
-        const currentAdapters = state.adapters ?? new Map()
-        const adapter = await getAdapter(localWalletType)
+        const currentAdapters = state.adapters ?? ({} as AdaptersByKeyManager)
+        const _adapter = await getAdapter(localWalletType)
 
-        if (adapter) {
+        if (_adapter) {
           try {
-            currentAdapters.set(localWalletType, [adapter])
+            // @ts-ignore TODO(gomes): FIXME, most likely borked because of WebUSBKeepKeyAdapter
```

Comment:
WebUSBKeepKeyAdapter types are too borked, fiddled for two hours in hdwallet with no success here, leaving this as-is for now - there is no effective runtime change here